ui/jobs/jobs.js

```diff
@@ -1563,6 +1564,73 @@ class Bars {
     };
   }
 
+  // (In Progress) (Need Help)
+  setupSch() {
+    let gcd = kUnknownGCD;
+
+    let BioBox = this.addProcBox({
+      id: 'sch-procs-bio',
+      fgColor: 'sch-color-bio',
+      scale: gcd,
+      threshold: gcd + 1,
+    });
+
+    let AetherflowBox = this.addProcBox({
+      id: 'sch-procs-aetherflow',
+      fgColor: 'sch-color-aetherflow',
+      scale: gcd,
+      threshold: gcd * 3, // you need at most 3 gcd to cast all 3 stacks out
+    });
+
+    let AetherflowRescourceBox = this.addResourceBox({
+      classList: ['sch-color-aetherflow'],
+    });
+
+    let LucidDreamingBox = this.addProcBox({
+      id: 'sch-procs-luciddreaming',
+      fgColor: 'sch-color-lucid',
+      scale: gcd,
+      threshold: gcd + 1,
+    });
+
+    this.jobFuncs.push((jobDetail) => {
+      let aetherflow = jobDetail.aetherflowStacks;
+      AetherflowRescourceBox.innerText = aetherflow;
+      AetherflowBox.threshold = gcd * aetherflow;
+
+      let p = AetherflowRescourceBox.parentNode;
+      if (aetherflow == 3 && AetherflowBox.duration <= 15) {
+        // then to red when under 15 seconds
+        // but you even have 3 aetherflow stacks
+        p.classList.add('need-to-throw');
+      } else {
+        p.classList.remove('need-to-throw');
+      }
+    });
+
+    this.abilityFuncMap[gLang.kAbility.Biolysis] = () => {
+      BioBox.duration = 0;
+      BioBox.duration = 30;
+    };
+    this.abilityFuncMap[gLang.kAbility.Aetherflow] = () => {
+      AetherflowBox.duration = 0;
+      AetherflowBox.duration = 60;
+    };
+    this.abilityFuncMap[gLang.kAbility.LucidDreaming] = () => {
+      LucidDreamingBox.duration = 0;
+      LucidDreamingBox.duration = 60;
+    };
+
+    this.statChangeFuncMap['SCH'] = () => {
+      gcd = this.gcdSpell();
```

Comment:
I don't think you need this line, since you're not using the gcd variable anywhere here.